README.md

```diff
@@ -186,6 +186,8 @@ Habitica is a gamified task manager, webapp and android/ios app, really wonderfu
 - [Vest](https://github.com/ealush/vest/labels/good%20first%20issue) _(label: good first issue)_ <br> Validations framework inspired by unit testing frameworks.
 - [Hoppscotch](https://github.com/hoppscotch/hoppscotch/labels/good%20first%20issue) _(label: good first issue)_ <br> A free, fast and beautiful API request builder.
 - [cypress](https://github.com/cypress-io/cypress/labels/first-timers-only) _(label: first-timers-only)_ <br>  Fast, easy and reliable testing for anything that runs in a browser.
+- [tinyhttp](https://github.com/talentlessguy/tinyhttp/labels/first-timers-only) _(label: first-timers-only)_ <br>  tinyhttp is a modern Express-like web framework written in TypeScript and compiled to native ESM, that uses a bare minimum amount of dependencies trying to avoid legacy hell.
```

Comment:
This label does not exist in the repo. Please remove the whole line as it will be covered by #825